deploy_website.sh

```diff
@@ -40,9 +40,13 @@ cp CONTRIBUTING.md docs/contributing.md
 # Build the site and push the new files up to GitHub
 mkdocs gh-deploy
 
+# Undo any file changes to be able to jump branches.
+git checkout -- .
+
 # Restore Javadocs from 2.x
 git checkout gh-pages
 git cherry-pick 0e1192aaa1d074c9703748fae100daef707218d4
+git push --set-upstream origin gh-pages
 git push
```

Comment:
combine these two lines?

Reply:
There is also a shorter version with `git push -u origin gh-pages` if you want!